src/presentation/controllers/ingest-controller.ts

```diff
@@ -0,0 +1,24 @@
+import { BaseController, PostRequest, RestController } from './base-controller'
+import { HttpErrorHandler } from '../interfaces/http-error-handler'
+import { Exception } from '../../model/exceptions/exception'
+import { Request, Response } from 'express'
+import { IngestService } from '../../business-logic/services/interfaces/ingest-service'
+
+@RestController('/ingest')
+export class IngestController extends BaseController {
+
+  constructor(private readonly ingestService: IngestService, private readonly httpErrorHandler: HttpErrorHandler) {
+    super()
+  }
+
+  @PostRequest('/reloadData/:rundownName')
+  public async reloadRundownData(req: Request, res: Response): Promise<void> {
+    try {
+      const rundownName: string = req.params.rundownName
+      await this.ingestService.reloadIngestData(rundownName)
+      res.send(`Attempted reload of rundown data for ${rundownName}`)
```

Comment:
Why 'attemped'?

Reply:
I was actually not sure what to say here, all we know is that we sent the signal to try and reload the data. We don't have a way of ensuring that happened, just that we "attempted" it.

Reply:
I have put a comment on the iNews gateway PR regarding, where i point out that the response should be sent after the reload/resync is done. If that is the case, then we know whether or not it has reloaded at this point.

Reply:
I have changed this to reflect that we have actually triggered reingest since we now know this with the latest changes to the iNews Gateway.